netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -304,6 +309,19 @@ private void cancelStream(ChannelHandlerContext ctx, CancelServerStreamCommand c
     encoder().writeRstStream(ctx, cmd.stream().id(), Http2Error.CANCEL.code(), promise);
   }
 
+  private void verifyContentType(int streamId, Http2Headers headers) throws Http2Exception {
+    ByteString contentType = headers.get(CONTENT_TYPE_HEADER);
+    if (contentType == null) {
+      throw Http2Exception.streamError(streamId, Http2Error.REFUSED_STREAM,
```

Comment:
Currently, this code is consistent with `determineMethod` ... if this needs to change, I think they both do.  WDYT? What should be the error? INTERNAL?